fdbserver/TLogServer.actor.cpp

```diff
@@ -2766,7 +2766,17 @@ ACTOR Future<Void> tLog( IKeyValueStore* persistentData, IDiskQueue* persistentQ
 		if(restoreFromDisk) {
 			wait( restorePersistentState( &self, locality, oldLog, recovered, tlogRequests ) );
 		} else {
```

Comment:
Out of curiosity, when will tLog not restoreFromDisk?
@alexmiller-apple I guess you know the answer? ;)

Reply:
When there's nothing to restore from on disk.  If a tlog crashed, then we end up here.  If we're recruiting a brand new tlog in a brand new process/disk, then we just initialize some files and carry on.